client/src/components/UserAvatar.tsx

```diff
@@ -6,6 +6,6 @@ function avatarUrl(user: User): string {
   return `https://cdn.discordapp.com/avatars/${user.discordId}/${user.discordAvatar}.png`
 }
 
-export default function UserAvatar(props: { user: User }) {
-  return <Avatar src={avatarUrl(props.user)} />
+export default function UserAvatar(props: { user: User; class?: string }) {
```

Comment:
**Improvement**
Instead of receiving a className as a way to control the sizing of the Avatar, have a few predefined sizes and operate only with those sizes, keeping the styling to make the sizing correct together with the `UserAvatar`.

**Reasoning**
As the codebase expands we might want to use the same size in multiple places, which would require us to either keep copies of the styles that give it that specific size (making it messy to update later, error prone, and making the file sizer bigger) _or_ we would put this size in a specific place and import it everywhere that used it. But then, if it's gonna stay in a central place, keeping it centralized directly with the component would be one of the best places possible.

The plan would be to call this component by:
```jsx
<UserAvatar size="small" user={someUser} />
```

or maybe even:
```jsx
<UserAvatar small user={someUser} />
```